src/components/HeaderSection.vue

```diff
@@ -4,17 +4,20 @@
       :showNavSearch="showNavSearch"
       :navSearchPlaceholder="navSearchPlaceholder"
     />
+    <Banner />
     <slot></slot>
   </header>
 </template>
 
 <script>
 import NavSection from '@/components/NavSection'
+import Banner from './Banner'
```

Comment:
Please rename the component to `DonationBanner` and update all references. `Banner` would be a good name if it was a re-usable component, but it's specifically for donating.

Reply:
Updated @zackkrida as u said😊